window.go

```diff
@@ -320,3 +320,17 @@ func SetWindowClosingHandled(handled bool) {
 func IsWindowClosingHandled() bool {
 	return ui.Get().Window().IsClosingHandled()
 }
+
+// Monitors returns the monitors reported by the system.
+func Monitors() []ui.Monitor {
```

Comment:
We should not expose `ui.Monitor` in the public APIs. Instead, we should have a struct `Monitor` wrapping `ui.Monitor`.

Reply:
This is what I thought as well. There'll have to be a semi-awkward conversion in the main ebiten package to convert from it for public consumption. I will merge ui.monitor with ui.Monitor functionality to allow this to happen.